src/components/user/event.tsx

```diff
@@ -101,7 +101,7 @@ export function Event({
           thread: renderThread(event.thread),
           comment: (
             <Link href={`/${event.comment.id}`} forceNoCSR>
-              {strings.entities.comment}
+              {`${strings.entities.comment} `}
```

Comment:
I believe [Best practice](https://stackoverflow.com/questions/40264084/best-practice-when-adding-whitespace-in-jsx) of adding spaces between two jsx elements would be to
- use `{' '}`
- put both elements in one line seperated by space (if prettier allows)
- in this case we could also use `&nbsp;` to make sure title and id are in the same line (non-breaking)

Reply:
&nbsp; seems very useful in this case!